src/core/xds/grpc/xds_cluster.cc

```diff
@@ -58,10 +58,15 @@ std::string XdsClusterResource::ToString() const {
                      lrs_backend_metric_propagation->AsString()));
   }
   if (use_http_connect) contents.push_back("use_http_connect=true");
-  if (!common_tls_context.Empty()) {
+  if (!upstream_tls_context.common_tls_context.Empty()) {
```

Comment:
Suggest adding a `ToString()` method on the `UpstreamTlsContext` struct, and then calling that from here.